### DISCUSSION TITLE
brew install doesn't work after running install script.

### DISCUSSION THREAD

**Discussion by FFILY** [Homebrew/discussions#840] [Everyday usage]
Hi, Experts:

It's the first time to use homebrew. I use macOS 11.2.1. But I don't know why I'm always getting the error message.
Please help me do the next to resolve it.
Appreciate! 
<img width="566" alt="Homebrew Error" src="https://user-images.githubusercontent.com/79067312/108202126-81070d00-70d5-11eb-9c3b-1882b33bad52.png">

**Comment by SMillerDev** [comment]
Please copy text next time, images are the worst for debugging tech issues.

---

Try running `brew update-reset` to reset the state of brew. It seems like some step failed when you ran the install script (see `fatal: could not resolve HEAD to a revision` in the output).

**Reply by FFILY** [reply]
I'm sorry, I don't know it. I will follow it.
Install Successfully.
Thank you!
